twisted/internet/test/test_socket.py

```diff
@@ -91,6 +96,7 @@ def test_invalidAddressFamily(self):
         reactor = self.buildReactor()
 
         port = socket.socket()
+        port.bind(("127.0.0.1", 0))
```

Comment:
It's necessary to `bind` before `listen`ing on Windows? If you call `listen(2)` without calling `bind` on a Un*x-like platform will have it bind to all interfaces, with a port chosen by the kernel (equivalent to `bind('0.0.0.0', 0)`)